src/openfl/net/NetStream.hx

```diff
@@ -404,6 +406,17 @@ class NetStream extends EventDispatcher {
 		
 		__playStatus ("NetStream.Play.seeking");
 		
+		if (__seekTimer != null) {
+			__seekTimer.stop();
+			__seekTimer = null;
+		}
+		__seekTimer = Timer.delay(() -> {
+			if (__seeking == true) {
+				__seeking = false;
+				__connection.dispatchEvent (new NetStatusEvent (NetStatusEvent.NET_STATUS, false, false, { code : "NetStream.Seek.Complete" } ));
+			}
+		}, 50);
+		
```

Comment:
What is the reason for adding a timer here? I suppose that seeking does not occur immediately, but I am concerned about adding a timer without connection to the state of the video. What if you request to seek, but you cannot (because it exceeds the video length) or the internet is slow and it is still buffering, or what if the video is stopped or seeked a second time before the timer fires?